### PR TITLE
Subscribers integration specific credentials

### PR DESCRIPTION
**Pull request by djabarovgeorge**

### CODE REVIEW

**Review by davidsoderberg — APPROVED** [review]
🏅

File: packages/node/src/lib/subscribers/subscribers.ts [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
-export class Subscribers {
+export class Subscribers implements ISubscribers {
   private readonly http: AxiosInstance;
 
   constructor(http: AxiosInstance) {
     this.http = http;
   }
 
-  async identify(subscriberId: string, data: ISubscriberPayload) {
+  async identify(subscriberId: string, data: ISubscriberPayload): Promise<any> {
```
Comment (scopsy) [inline packages/node/src/lib/subscribers/subscribers.ts:12]: Let's added the returned type here instead of using any.

Comment (scopsy) [reply packages/node/src/lib/subscribers/subscribers.ts:12]: Sure totally understand you, could you create a discussion for this?

Comment (djabarovgeorge) [reply packages/node/src/lib/subscribers/subscribers.ts:12]: Sure i will create one shortly :)

Comment (djabarovgeorge) [reply packages/node/src/lib/subscribers/subscribers.ts:12]: https://github.com/novuhq/novu/discussions/592

Comment (djabarovgeorge) [reply packages/node/src/lib/subscribers/subscribers.ts:12]: Added https://github.com/novuhq/novu/discussions/592

**Pull request merged**